rest/endpoints/media/media_playback_sessions.yaml

```diff
@@ -143,6 +143,92 @@ paths:
                     on another device in order to watch on this one.
 
 
+# ------------------------------------------------------------------------------------------------------------------------
+  
+  /users/{user_id}/media_play_sessions/{session_id}:
+
+    put:
+      tags:
+      - Media Play Sessions
+      summary: Update MediaPlaySession for a User. 
```

Comment:
the endpoint can not only update, but also create if the ID does not exist
for more info: https://jira.aminocom.com/browse/BPLAT-11251